src/phoenix/server/app.py

```diff
@@ -26,22 +28,60 @@
 
 logger = logging.getLogger(__name__)
 
+templates = Jinja2Templates(directory=SERVER_DIR / "templates")
+
+
+@dataclass(frozen=True)
+class Config:
+    host: str
+    port: int
+    has_corpus: bool
+
 
 class Static(StaticFiles):
     "Static file serving with a fallback to index.html"
 
+    _config: Config
+
+    def __init__(
+        self,
+        *,
+        directory: Optional[PathLike] = None,
+        packages: Optional[List[Union[str, Tuple[str, str]]]] = None,
+        html: bool = False,
+        check_dir: bool = True,
+        follow_symlink: bool = False,
+        config: Config,
+    ):
+        self._config = config
+        super().__init__(
+            directory=directory,
+            packages=packages,
+            html=html,
+            check_dir=check_dir,
+            follow_symlink=follow_symlink,
+        )
```

Comment:
maybe we can leave these other parameters anonymous since they are not from us
```suggestion
    def __init__(self, *, config: Config, **kwargs: Any):
        self._config = config
        super().__init__(**kwargs)
```